app/helpers/elmo_form_builder.rb

```diff
@@ -147,6 +147,9 @@ def elmo_field(field_name, options)
             ""
           end
 
+        when :file
+          options[:value].original_filename
```

Comment:
Getting closer. Try `val` instead of `options[:value]`.

Think about how say, a regular text field works. You don't have to specify a `:value` option. Note that `val` above gets the existing value of the attribute using `send`. Try to look for parallels like this when you're refactoring something.